DependencyInjection/Configuration.php

```diff
@@ -97,7 +97,7 @@ private function addDbalSection(ArrayNodeDefinition $node)
                             ->end()
                             ->children()
                                 ->scalarNode('class')->isRequired()->end()
-                                ->booleanNode('commented')->defaultTrue()->end()
+                                ->booleanNode('commented')->defaultFalse()->end()
```

Comment:
This is a BC break...

Reply:
For the first part, if it helps, I think this is an example: https://github.com/symfony/symfony/blob/2e47edc4a52c3b61664e512f849c3df5f5c20593/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L714-L721

It's checking to see if `strict_email` is explicitly configured, and it triggers a deprecation warning if it is.

For the second part... I'm not as sure :)